test/iidm/converter/xml/TerminalRefTest.cpp

```diff
@@ -24,6 +27,11 @@ BOOST_FIXTURE_TEST_CASE(TerminalRef, test::ResourceFixture) {
     test::converter::RoundTrip::roundTripVersionedXmlTest("terminalRef.xml", IidmXmlVersion::all());
 }
 
+BOOST_FIXTURE_TEST_CASE(TerminalRefNotFoundTest, test::ResourceFixture) {
+    const auto& filePath = test::converter::RoundTrip::getVersionedNetworkPath("terminalRefNotFound.xml", IidmXmlVersion::V1_5());
+    POWSYBL_ASSERT_THROW(Network::readXml(filePath), PowsyblException, "Terminal reference identifiable not found: '\?\?\?\?'");
```

Comment:
Notice : I escaped "????" because I had the strange following error :  
````
/mnt/e/Projet_SRO/rte/powsybl-iidm4cpp-master/test/iidm/converter/xml/TerminalRefTest.cpp:32:119: error: trigraph ignored [clang-diagnostic-trigraphs]
    POWSYBL_ASSERT_THROW(Network::readXml(filePath), PowsyblException, "Terminal reference identifiable not found: '????'");
                                                                                                                      ^
16712 warnings and 1 error generated.
```

Some helper here : https://webdevdesigner.com/q/purpose-of-trigraph-sequences-in-c-7706/

Reply:
Thanks to you, I learn something today! Maybe the best option is to replace the `????` by another ID in the XML file?

Reply:
Agree, I replaced it by 'UNKNOWN'